clio-client/src/main/scala/org/broadinstitute/clio/client/ClioClient.scala

```diff
@@ -25,10 +25,9 @@ object ClioClient extends App {
 
 class ClioClient(val webClient: ClioWebClient) {
 
-  val parser = new BaseParser
-
   def execute(args: Array[String])(implicit ec: ExecutionContext): Int = {
-    val success = parser.parse(args, BaseArgs()) match {
+    val parser: BaseParser = new BaseParser
```

Comment:
We don't, except perhaps in a test. I was worried that the change was required due to a test, in which case I wanted to know if the parser wasn't actually immutable.

I guess on the other hand, this limits the scope of the object to area where it's needed.